client/src/modules/Job/pages/NoAuthLanding/components/EmployerInfo.tsx

```diff
@@ -0,0 +1,120 @@
+import { Button } from 'antd';
+import { COLORS } from 'modules/Job/theme/colors';
+
+export const EmployerInfo = () => {
+  return (
+    <>
+      <div className="wrapper">
```

Comment:
Same as above regarding usage of Antd components instead of raw html and styles

Reply:
there is a problem here, in order to implement responsive layout I need media query in css, and I can't implement it with Antd.

Reply:
here example
https://ant.design/components/grid#components-grid-demo-gutter

Reply:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/57412523/221233659-185054bb-e04c-473e-938c-22fc916d54aa.png)

Reply:
i need media query here for responsive padding